tools/webdriver/webdriver/client.py

```diff
@@ -370,8 +370,8 @@ def __init__(self,
         self.actions = Actions(self)
 
     def __eq__(self, other):
-        return (self.session_id is not None and isinstance(other, Session)
-                and self.session_Id == other.session_id)
```

Comment:
Note also the typo here, `self.session_Id`. If that part were reached it would throw an exception, so there can't be any tests for this.